.gitignore

```diff
@@ -260,5 +260,9 @@ launchSettings.json
 /coverages
 **/.DS_Store
 
+# UT coverage report
+/coverages
+**/.DS_Store
+
```

Comment:
this is a dupe of the above lines. please remove it.

Reply:
> this is a dupe of the above lines. please remove it.

I’ll remove it later